openedx/frontend-app-authoring
# feat: Add tags to info sidebar of library component [FC-0062]

## Description

This:
- Adds the `drawer` (default) and `component` variants for `ContentTagsDrawer`.
- Adds `ContentTagsDawer` in the info sidebar of library components.
- Which edX user roles will this change impact? "Course Author"

![image](https://github.com/user-attachments/assets/915ea141-6223-4e09-ae1b-b7f3149863ae)

## Supporting information

Github Issue: https://github.com/openedx/frontend-app-authoring/issues/1286
Internal ticket: [FAL-3847](https://tasks.opencraft.com/browse/FAL-3847)

## Testing instructions

- Run this script to add taxonomies and tags: https://github.com/open-craft/taxonomy-sample-data
- Go to studio and create a new library if you don't have one created.
- Go to the library home of your library.
- Create a new component on your library
- Click on the component card to open the component info. Go to `Manage` tab.
- Verify that the `ContentTagsDrawer` is rendered. Click on `Manage tags` and add some tags. Save.
- Verify that the tag count on the component card and the component info sidebar is updated.
- Play with the `ContentTagsDrawer` (add more tags, delete tags, etc)
